### PR TITLE
chore(styling): improve styling based on ux feedback

### PR DESCRIPTION
**Pull request by seanforyou23**
This PR applies a few styling enhancements based on a recent UX review.

- Add new status to the integration details page
- Fix clipboard copy button padding
- Consolidate header menu item (logout) into single menu at small viewports
- Add stub avatar
- Header background again expands full width on small viewports

Should help close https://github.com/syndesisio/syndesis/issues/5535

Screen Shot 2019-06-03 at 8 41 44 PM

Screen Shot 2019-06-03 at 8 41 59 PM

Screen Shot 2019-05-24 at 3 42 17 PM

Screen Shot 2019-05-31 at 11 31 55 AM

better spacing strategy for title
add status to integration details view
fix clipboard copy btn styling issue
fix header background issue
update modal/logo image assets for productized/upstream build
update to latest patternfly versions
give duration control an id
fix font sizes on dashboard (body content should generally be 14px)
remove bolding from integration list item titles
replace selective base patternfly stylesheets (this fixes several global font issues)

### CODE REVIEW

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1836/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

File: app/ui-react/packages/ui/src/Integration/Details/IntegrationDetailInfo.tsx [hunk 1]
```diff
@@ -24,29 +25,32 @@ export class IntegrationDetailInfo extends React.PureComponent<
...
+            )}
+            {this.props.currentState === 'Published' && this.props.version && (
+              <>
+                <span className="pficon pficon-ok" />
+                &nbsp;Published version {this.props.version}
+              </>
+            )}
+          </>
+        </div>
+        {this.props.status}
```
Comment (gashcrumb) [inline app/ui-react/packages/ui/src/Integration/Details/IntegrationDetailInfo.tsx]: This doesn't really look good when the integration is running:

image

We don't show this status in the angular app:

image

Comment (gashcrumb) [reply app/ui-react/packages/ui/src/Integration/Details/IntegrationDetailInfo.tsx]: This was here already:

image

This is the missing bit:

image

File: app/ui-react/packages/auto-form/src/widgets/FormDurationComponent.tsx [hunk 2]
```diff
@@ -101,6 +101,7 @@ export class FormDurationComponent extends React.Component<
     );
   }
   public render() {
+    const controlId = `${toValidHtmlId(this.props.field.name)}-duration`;
```
Comment (gashcrumb) [inline app/ui-react/packages/auto-form/src/widgets/FormDurationComponent.tsx]: Ah good call, thanks!

File: app/ui-react/syndesis/src/modules/integrations/components/IntegrationDetailHeader.tsx [hunk 3]
```diff
@@ -16,6 +16,7 @@ import {
   IntegrationDetailBreadcrumb,
   IntegrationDetailEditableName,
   IntegrationDetailInfo,
+  // IntegrationStatus,
```
Comment (gashcrumb) [inline app/ui-react/syndesis/src/modules/integrations/components/IntegrationDetailHeader.tsx]: U should delete this.

File: app/ui-react/syndesis/src/index.css [hunk 4]
```diff
@@ -1,4 +1,5 @@
 body {
+  display: flex;
```
Comment (gashcrumb) [inline app/ui-react/syndesis/src/index.css]: I feel like this line should scare me a little bit, I dunno why.  Am suspicious.

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/1962/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

File: app/ui-react/syndesis/src/index.css (hunk 4 above)
Comment (seanforyou23) [reply app/ui-react/syndesis/src/index.css]: So I added this to fix an issue where the background color of the header doesn't fit the full width of the viewport on really small screen sizes. Can remove, but seems like we might want it until the issue is fixed on PF side.

Comment (riccardo-forina) [reply app/ui-react/syndesis/src/index.css]: I'm personally a +1 for waiting for the official fix.

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/2040/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

File: app/ui-react/syndesis/src/index.css (hunk 4 above)
Comment (seanforyou23) [reply app/ui-react/syndesis/src/index.css]: Cool, I'm pretty sure is a fudge between pf3/pf4 - will look into it. Keep ya posted!

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/2073/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

File: app/ui-react/syndesis/src/index.css (hunk 4 above)
Comment (mcoker) [reply app/ui-react/syndesis/src/index.css]: Looks like the issue is that the user menu in the masthead makes the header too wide. We move the user menu (and other masthead icons/menus) into a kebab on mobile in pf4. You can see that general behavior here - https://pf4.patternfly.org/demos/Page/examples-full/?component=Page%20component%20default%20nav%20header%20example

If we change the user menu dropdown to an icon or something more narrow, the header won't overflow the x-axis of the page, and `display: flex` shouldn't be needed.

**Comment by riccardo-forina** [comment]
PR Storybook available [here](https://circleci.com/api/v1.1/project/github/syndesisio/syndesis-react/2116/artifacts/0/home/circleci/repo/app/ui-react/doc/index.html)

**Comment by riccardo-forina** [comment]
Was checking the PR and I wondered, is that the proper icon and placement for the dropdown?

image

**Comment by seanforyou23** [comment]
@riccardo-forina no, it's still WIP. The interactivity is the main thing I was focusing on in yesterday's push. Today I will be focusing on cleanup like fixing the exact icons and renaming some of the image assets to be more consistent. Looks like the toolbar layout is what's creating too much space there, I'll address that also.

Let me know if you see any problems with the rest of the implementation in the meantime.

**Pull request merged**